soccer/gameplay/tactics/positions/submissive_defender.py

```diff
@@ -13,11 +13,13 @@
 # The regular defender does a lot of calculations and figures out where it should be
 # This defender lets someone else (the Defense tactic) handle calculations and blocks things based on that
 class SubmissiveDefender(
-        single_robot_composite_behavior.SingleRobotCompositeBehavior):
+    single_robot_composite_behavior.SingleRobotCompositeBehavior):
     class State(Enum):
         ## gets between a particular opponent and the goal.  stays closer to the goal
         marking = 1
-        # TODO: add clear state to get and kick a free ball
+        clearing = 2
+
+    go_clear = False
```

Comment:
Putting this declaration here makes it a class variable - shared between all instances of SubmissiveDefender.  If you put it inside __init__ as `self.go_clear = False`, it will be an instance variable which is probably what you want.